### ISSUE TITLE
Examples using standard MicroPython

### ISSUE THREAD

**Issue by kokujin** [mozilla-iot/webthing-upy#23]
Is there some way to get web things running using the standard MicroPython version? They present way described is tiresome and eventually caused problems.

It would be awesome if this repository had a pre-built MicroPython version with all the necessary modules that one can download and test.

Thanks

**Comment by hoihu** [comment]
yes I had similar experiences and raised https://github.com/mozilla-iot/webthing-upy/issues/21

**Comment by rzr** [comment]
I would be more in favor to script the rebuild of the engine than ship binaries in sources
